/*
Throw, and Try...Catch...Finally: 

The 'try' statement defines a code block to run (to try).

The 'catch' statement defines a code block to handle any error.

The 'finally' statement defines a code block to run regardless of the result.

The 'throw' statement defines a custom error.
*/

/*
Syntax: 
try {
  Block of code to try
}
catch(err) {
  Block of code to handle errors
}
finally {
  Block of code to be executed regardless of the try / catch result
}
*/